src/tests/tests_runtime.rs

```diff
@@ -11,6 +11,18 @@ use serial_test::serial;
 use std::sync::Arc;
 use wasmer::WasmPtr;
 
+#[test]
+#[serial]
+fn test_expr() {
```

Comment:
This uses gas calibration test in runtime??

Reply:
Was a WIP, I added 2 security tests with the corresponding WAT files